src/helpers/assets.js

```diff
@@ -77,7 +77,7 @@ const addEthPlaceholder = (
       price: value,
       symbol: 'ETH',
       type: 'token',
-      uniqueId: 'eth',
+      uniqueId: 'eth_mainnet',
```

Comment:
im sorry i set u up. now that we have optional network for mainnet, this can go back to being just eth